On The Map/Controller/MapKitViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  MapKitViewController.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/25/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class MapKitViewController: UIViewController, MKMapViewDelegate {
+    
+    var students:[Student] = []
+
+    @IBOutlet weak var mapView: MKMapView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mapView.delegate = self
+        self.pinLocatinDisplay()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
+        Client.getStudentsLocation { (response, error) in
+            guard let response = response, error == nil else {
+                return
+            }
+            self.refetchLocations(response: response)
+        }
+    }
+    
+    @IBAction func refreshStudentLocation(_ sender: Any) {
+        Client.getStudentsLocation { (response, error) in
+            guard let response = response, error == nil else {
+                return
+            }
+            self.refetchLocations(response: response)
+        }
+    }
```

Comment:
these two functions seem to do the exact same thing, so it might make sense to separate that out into its own function and then call it from both places

Reply:
Yeah good call, I should have abstracted it.